### DISCUSSION TITLE
I have lost access to my main GH account, 2F. How to receive real assistance?

### DISCUSSION THREAD

**Discussion by BandyALVL** [community/community#143291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Greetings.

On my main account I have enabled 2F, stored the codes.
In the process of switching the laptops I moved the codes to the hard drive.
Archive with it appeared to be broken. Any ways to restore the access?

**Comment by MasterInQuestion** [comment]
What did you do with the "archive" procedure?
    If you cannot restore the "codes" it's likely you would lose the account, forever.
